### PR TITLE
Dynamically retrieve container ID

### PR DESCRIPTION
**Pull request by Btlyons1**
- Updated createTagLayer
- Grabs the ID from the Docker API ImageHistory method and assigns it to baseLayer

### CODE REVIEW

**Comment by josephschorr** [comment]
@Btlyons1 Sorry, just saw this. Mind rebasing and then I'll review? Apologies for the insanely delayed review...

**Comment by Btlyons1** [comment]
Hey @josephschorr no worries. I created the PR for @gamkiller77. If you like the added functionality of grabbing the baselayer dynamically instead of passing a flag, I'm more than willing to rebase. If not, you can close the PR.

**Comment by josephschorr** [comment]
@Btlyons1 Yeah, definitely. I think it is worth doing.

**Comment by Btlyons1** [comment]
@gamkiller77 Mind giving these changes a test? 
@josephschorr I moved the dynamic assignment of the baseLayer variable to line 441 where it checks if the baseLayer/baseImage vars are empty. If those values are empty, it will assign baseLayer dynamically to the first image ID in the image history. I didn't want to change the functionality that Kenny's PR introduced. If you don't agree with the changes, let me know. Thanks

File: monitor.go [hunk 1]
```diff
@@ -439,7 +439,12 @@ func main() {
 	}
 
 	if *baseImage == "" && *baseLayer == "" {
-		log.Fatalln("Missing base-image and base-layer-id flag; only one of required")
+		log.Infof("Missing base-image and base-layer-id flag; Dynamically assinging base-layer")
+		grabID, err := dockerClient.ImageHistory(*repository)
+		if err != nil {
+			log.Fatalln("Failed grab image ID: %v", err)
+		}
+		*baseLayer = grabID[0]
```
Comment (josephschorr) [inline monitor.go]: Add a line logging the base ID grabbed

**Pull request merged**